vernac/vernacentries.ml

```diff
@@ -2308,8 +2308,7 @@ let with_fail ~st f =
     | HasNotFailed as e -> raise e
     | e when CErrors.noncritical e || e = Timeout ->
```

Comment:
I guess we should add an `is_anomaly` check here?

Reply:
>I guess we should add an is_anomaly check here?

I dunno. What behaviour difference would we see?

Reply:
Yeah not a lot, once we have something such as registration for errors this case would not ever happen.